boxcli/plan.go

```diff
@@ -20,46 +15,15 @@ func PlanCmd() *cobra.Command {
 	flags := planCmdFlags{}
 
 	command := &cobra.Command{
-		Use:   "plan",
-		Short: "Preview the plan used to build your environment",
-		Args:  cobra.MaximumNArgs(1),
+		Use:        "plan",
+		Short:      "Preview the plan used to build your environment",
+		Deprecated: "Please follow devbox documentation on how to build a container image around your devbox project.",
+		Args:       cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runPlanCmd(cmd, args, flags)
+			return nil
```

Comment:
`plan` command is good for testing shell stuff as well. Can we simply hide instead of removing it?

Reply:
Fair! I didn't know it's used for testing. I'll update the PR.

Reply:
@LucilleH This pr is too convoluted to keep `plan` and remove the build. I'll close it and open a new PR in a different branch